cobc/scanner.l

```diff
@@ -1257,11 +1257,189 @@ error_literal (const char *type, const char *literal)
 	cb_error ("%s", err_msg);
 }
 
+#ifndef COB_EBCDIC_MACHINE
```

Comment:
Don't we have a conceptional issue here?
EBCDIC on zLinux ("in the cloud", that's the most likely EBCDIC you'd move an old program to) won't match GCOS EBCDIC expectations, would it?

Reply:
`COB_EBCDIC_MACHINE` is probably not precise enough. Could it then be worth using a predefined set of values for a variable like `COB_MACHINE_CHARS`, with, for now, `ZLINUX` (kind of IBM EBCDIC, if there's only one), and (ASCII8)?